### ISSUE TITLE
page rendering is slow

### ISSUE THREAD

**Issue by greyltc** [l3iggs/docker-owncloud#62]

**Comment by elangelo** [comment]
right now, all i can add is that httpd cpu usage is high for every request.

**Comment by greyltc** [comment]
I can't recreate this; my server seems to be quite snappy. I don't think this is an issue with the container. I'll need to see some logs with errors or something other than "it's slow." To be able to address this.

**Comment by nucle** [comment]
Hi, first of all thank you for this great container.
Page rendering is really slow. Somtimes it takes up to 20 seconds.

What can i do?

Here are some logs. probably it can help:

Warning core    Login failed: 'root' (Remote IP: '172.17.0.1')  2016-02-19T21:34:24+00:00
Warning core    Login failed: 'root' (Remote IP: '172.17.0.1')  2016-02-19T21:33:22+00:00
Error   PHP Array to string conversion at /usr/share/webapps/owncloud/lib/private/template/functions.php#36 2016-02-19T01:12:34+00:00
Error   PHP Array to string conversion at /usr/share/webapps/owncloud/lib/private/template/functions.php#36 2016-02-19T01:10:13+00:00
Warning core    Login failed: 'root' (Remote IP: '172.17.0.1')  2016-02-19T01:08:38+00:00
Warning core    Login failed: 'root' (Remote IP: '172.17.0.1')  2016-02-19T01:08:18+00:00
Warning core    Login failed: 'root' (Remote IP: '172.17.0.1')  2016-02-19T01:08:08+00:00
Error   PHP Array to string conversion at /usr/share/webapps/owncloud/lib/private/template/functions.php#36 2016-02-19T01:02:37+00:00
Error   PHP is_dir(): open_basedir restriction in effect. File(/usr/share/pear/) is not within the allowed path(s): (/srv/http/:/dev/urandom:/tmp/:/usr/share/pear/:/usr/share/webapps/owncloud/:/etc/webapps/owncloud:/dev/urandom) at /usr/share/webapps/owncloud/3rdparty/symfony/process/Symfony/Component/Process/ExecutableFinder.php#59

**Comment by peperunas** [comment]
+1 having the same issue

**Comment by greyltc** [comment]
Could any of you seeing this issue try a few things to test if I've fixed it please?

First, please try pulling the latest image:

```
docker pull l3iggs/owncloud
docker run --name oc -p 80:80 -p 443:443 -d l3iggs/owncloud
```

and let me know if it's fixed there.
If it's not fixed there, please try pulling the dev tag and let me know if it's fixed there:

```
docker pull l3iggs/owncloud:dev
docker run --name oc -p 80:80 -p 443:443 -d l3iggs/owncloud:dev
```

Thanks! Looking forward to hearing some feedback!

**Comment by elangelo** [comment]
Both these images result in HTTP 500's ...

I've been trying to locate some logs but i'm not finding anything :s. This is probably pebkac, so if you tell me where to look....

**Comment by greyltc** [comment]
But I bet it renders fast now! Rendering speed issue fixed!

**Comment by greyltc** [comment]
@elangelo Seriously: I just checked and I'm not seeing error 500 with either of these images. Could your 500 errors be some mistake on your end? are you running exactly the commands I pasted?

**Comment by greyltc** [comment]
I'm going to close this now. Other users have recently reported that page rendering is not slow for them. So the issue that page rendering is generically slow across the board seems invalid. Maybe even the newly released ownCloud version 9.0 magically solves this.

There might be some specific configuration that is causing this (such as using ldap maybe?) If you can give me instructions on how to recreate this, let's re-open it with a more specific bug report (like ldap causes slow page rendering) or (page rendering when browsing with the Safari browser is slow).
